### PR TITLE
add collective precompile docs

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR adds docs on the Collective precompile

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [ ] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

n/a

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [x] No additional PRs are required after the translations are done

#### Items to be Updated

n/a

### CODE REVIEW

**Review by albertov19 — CHANGES_REQUESTED** [review]
Looks good! My only comment/doubt is why a proposal has to be approved first and then voted on by the collective. For example, is that the case for Treasury proposals? If so, our "happy path" in the treasury page does not reflect that.

I would add an explainer of around the different steps or something similar.

File: builders/pallets-precompiles/precompiles/collective.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+2. Select the **proposals** extrinsic
+3. Toggle the **include option** slider off
+4. Click **+** to submit the query
+5. The results will appear below with the proposal index and the proposal details
+
+![View all treasury proposals](/images/builders/pallets-precompiles/precompiles/collective/collective-5.png)
+
+Now that you have the proposal and the proposal index, you'll be able to approve the proposal in the following section using the collective precompile.
+
+### Propose the Proposal {: #propose-te-proposal }
```
Comment (eshaben) [inline builders/pallets-precompiles/precompiles/collective.md:142]:
```suggestion
### Propose the Proposal {: #propose-the-proposal }
```

**Comment by eshaben** [comment]
@albertov19 Olivia said the treasury page needs to be updated and she was looking at it similarly to the governance page. Not sure if we should just wait for the updates she is requesting or I can try to look at it and fix them but seems like duplicated efforts

What is described on the treasury page seems to reflect what I have here though:

```
Once a proposal has been submitted, the treasury council votes on it. The threshold for accepting a treasury proposal is at least three-fifths of the treasury council. On the other hand, the threshold for rejecting a proposal is at least one-half of the treasury council. Please note that there is no way for a user to revoke a treasury proposal after it has been submitted.

If approved by the treasury council, the proposal enters a queue to be placed into a spend period. If the spending queue happens to contain the number of maximum approved proposals, the proposal submission will fail similarly to how it would if the proposer's balance is too low. If the proposal gets rejected, the deposit will be lost and transferred to the parachain treasury account.
```

So I think the happy path diagram is correct unless I am missing something haha which could very well be the case

**Comment by eshaben** [comment]
Changes will be made in a separate PR for treasury updates

**Review by albertov19 — APPROVED** [review]

**Pull request merged**